cmd/protoc-gen-openapi/generator/openapi-v3.go

```diff
@@ -315,7 +316,16 @@ func getMessageName(message protoreflect.MessageDescriptor) string {
 }
 
 func (g *OpenAPIv3Generator) formatMessageName(message *protogen.Message) string {
+	typeName := fullMessageTypeName(message.Desc)
+
 	name := getMessageName(message.Desc)
+	if !*g.conf.FQSchemaNaming {
```

Comment:
This is also a question of: should we be consistent or backwards compatible.
I would personally prefer that types are the same, regardless of the naming.
I would be surprised if I changed the naming option and got different types.

Reply:
I'm not sure about the history of the naming here, but without it being fully qualified with the package prefix then the name of the schema would just be `Value` which seems a little too general perhaps. Same with `Any`. I am happy to switch them both to just be `Value` and `Any` in the case non-fully-qualified names if that is what you prefer. I personally will always use the fully qualified names so it doesn't matter to me. :)

Reply:
Aahh... Good point! You're absolutely right 👍 It would easily risk conflicting with a user's naming, which is why the  `AnyJSONValue` name was chosen in the first place, if I remember correctly.

Reply:
Would it make more sense to have `GoogleProtobufValue` instead of `AnyJSONValue`?

Reply:
If you are ok with a possibly breaking change, that is fine with me. I'll do what the repo owners here want. :)

Reply:
:) It makes sense to me, as one of the overall goals is to be able to convert both ways, so to have the "full" name, is probably more sane, than  `AnyJSONValue`. What are your thoughts @timburks?

Reply:
No strong opinion, but `GoogleProtobufValue` sounds more specific. I'm fine with whatever you two work out.